.travis.yml

```diff
@@ -28,20 +28,32 @@ jobs:
       env: CXX=clang++ PYTHON=python3 CXXFLAGS=-std=c++98
     - os: linux
       env: CXX=clang++ PYTHON=python3 CXXFLAGS=-std=c++11
+    - os: linux
+      env: CXX=g++ PYTHON=pypy CXXFLAGS=-std=c++11
+    - os: linux
+      env: CXX=clang++ PYTHON=pypy CXXFLAGS=-std=c++98
```

Comment:
I'm not convinced such extensive coverage is needed. I'd only add two new build variants to cover pypy and pypy3 with C++11.

Reply:
I'll drop those and re-push.